src/physics/utilities/boundary_condition.hpp

```diff
@@ -191,6 +191,35 @@ class BoundaryCondition {
    */
   void projectBdr(const double time, const bool should_be_scalar = true) const;
 
+  /**
+   * @brief Projects the boundary condition over boundary to a DoF vector
+   * @param[in] time The time for the coefficient, used for time-varying coefficients
+   * @param[in] should_be_scalar Whether the boundary condition coefficient should be a scalar coef
+   * @pre A corresponding field (FiniteElementState) has been associated
+   * with the calling object via BoundaryCondition::setTrueDofs(FiniteElementState&)
+   */
+  void projectBdrToDofs(mfem::Vector& dof_values, const double time, const bool should_be_scalar = true) const;
+
+  /**
+   * @brief Allocates an integrator of type "Integrator" on the heap,
+   * constructing it with the boundary condition's vector coefficient,
+   * intended to be passed to mfem::*LinearForm::Add*Integrator
+   * @return An owning pointer to the new integrator
+   * @pre Requires Integrator::Integrator(mfem::VectorCoefficient&)
+   */
+  template <typename Integrator>
+  std::unique_ptr<Integrator> newVecIntegrator() const;
+
+  /**
+   * @brief Allocates an integrator of type "Integrator" on the heap,
+   * constructing it with the boundary condition's coefficient,
+   * intended to be passed to mfem::*LinearForm::Add*Integrator
+   * @return An owning pointer to the new integrator
+   * @pre Requires Integrator::Integrator(mfem::Coefficient&)
+   */
+  template <typename Integrator>
+  std::unique_ptr<Integrator> newIntegrator() const;
+
```

Comment:
These were removed a little while back - it doesn't look like they're used anywhere so this could be from a merge that didn't go quite right.